lib/fork.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var childProcess = require('child_process');
+var Promise = require('bluebird');
+var assign = require('object-assign');
+var join = require('path').join;
+
+module.exports = fork;
+
+function fork(file) {
+	var babel = join(__dirname, 'babel.js');
+
+	var options = {
+		silent: true,
+		env: assign({}, process.env, {AVA_FORK: 1})
+	};
+
+	var ps = childProcess.fork(babel, [file], options);
+
+	ps.stdout.pipe(process.stdout);
+	ps.stderr.pipe(process.stderr);
```

Comment:
```js
ps.stdout.pipe(process.stdout);
ps.stderr.pipe(process.stderr);
```

This is not for sending data from forked processes. I am sending data via `process.send()`. This is for piping output from forked processes, so that `console.log()`s won't be hidden, when executed via CLI.